### PR TITLE
Level overview screen

### LINKED ISSUE #3: Start Creating the level overview screen
This will include its associate data such as the lesson titles and a description.

### PR DESCRIPTION
**Pull request by markusamuel**

### CODE REVIEW

File: src/consts/lessons.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const DATA_LEVEL_OVERVIEW = [
+  {title: 'Level One', data: ['Lesson 1', 'Lesson 2', 'Lesson 3']},
+];
```
Comment (markusamuel) [inline src/consts/lessons.js]: Dummy Data

File: src/App.js [hunk 2]
```diff
@@ -4,10 +4,11 @@ import {createDrawerNavigator} from '@react-navigation/drawer';
 import {NavigationContainer} from '@react-navigation/native';
 
 import LessonBase from './screens/lessonBase';
+import LevelOverview from './screens/levelOverview';
 
 const HomeScreen = ({navigation}) => (
   <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
-    <Button onPress={() => navigation.navigate('Level One')} title='Go to Level One' />
+    <LevelOverview />
```
Comment (mark-yacoub) [inline src/App.js]: actually, we wanna keep the home page, but replace level1-4 by the level overview page. you see it when you go to a level, and each level overview takes you to the lesson of that level

File: src/screens/levelOverview.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {SectionList, StyleSheet, Text, View} from 'react-native';
+import {DATA_LEVEL_OVERVIEW} from '../consts/lessons';
+
+const styles = StyleSheet.create({
```
Comment (mark-yacoub) [inline src/screens/levelOverview.js]: styles usually live at the bottom, right before export

File: src/screens/levelOverview.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {SectionList, StyleSheet, Text, View} from 'react-native';
+import {DATA_LEVEL_OVERVIEW} from '../consts/lessons';
```
Comment (mark-yacoub) [inline src/screens/levelOverview.js]: let's not have a space between all the react-* imports, then 1 space before utils/helpers/consts/...

**Pull request closed without merging**